crates/bevy_pbr/src/atmosphere/resources.rs

```diff
@@ -369,7 +372,7 @@ impl SpecializedRenderPipeline for RenderSkyBindGroupLayouts {
             },
             zero_initialize_workgroup_memory: false,
             fragment: Some(FragmentState {
-                shader: shaders::RENDER_SKY.clone(),
+                shader: self.shader.clone(),
```

Comment:
Why was this one handled differently?

Reply:
This one builds its `RenderPipelineDescriptor` in the `SpecializedRenderPipeline` impl. The others build it in their FromWorld impl where they have access to the asset server.